### DISCUSSION TITLE
"404: not found" error on SolidJS project deployed to Vercel

### DISCUSSION THREAD

**Discussion by harryalbert** [vercel/community#2405] [Help] [answered]
### Summary

I recently created a personal website using SolidJS, with server side rendering and typescript enabled. I attempted to deploy this site to Vecel, with the only change from the base vercel settings being that I changed the Output Directory to 'dist'. However, after deploying the site I am getting a 404 error. I do not get this error when running 'vercel dev' locally. The error only occurs on my actual Vercel site. You can find the uploaded site (and the error) [here](https://personal-website-ten-sooty.vercel.app/), and the github repo for my solidjs project [here](https://github.com/harryalbert/personal_website). Does anyone have any advice on how to fix this issue? Thank you!
<img width="917" alt="Screen Shot 2023-05-10 at 10 46 54 AM" src="https://github.com/vercel/community/assets/65182701/50a5a6a5-29ee-47a3-8917-643c4000eb1a">

**Comment by harryalbert** [comment, marked as answer]
The answer was to change my vite config to plugins to
'''
plugins: [solid({adapter: vercel()})],
'''

**Comment by AuroraLantean** [comment]
adapter is no longer needed. See my solution:
https://stackoverflow.com/questions/76219797/404-not-found-error-on-solidjs-project-deployed-to-vercel/78677704#78677704
